### ISSUE TITLE
Filter

### ISSUE THREAD

**Issue by winnidat** [Wunderbyte-GmbH/moodle-mod_newsletter#45]
Filter adds current moodle Website if a link in the newsletter does not start with https://www.

**Comment by michaelpollak** [comment]
Thank you for the report, can you give me some more details.
If I understand you correctly, you are trying to send a newsletter issue.
You add an URL, let's say http://www.thisisatest.org
You send the issue.
Your receivers see a different URL?

**Comment by dasistwas** [comment]
@winnidat Steps to reproduce would be nice.

**Comment by dasistwas** [comment]
the problem is with urls not starting with www. like http://wunderbyte.at, then the link is tansformed into: http://moodlroot.com/http://wunderbyte.at

**Comment by michaelpollak** [comment]
I've just installed the most recent version and can't confirm what you are seeing.
If I set up an url wunderbyte.at the system links to http://moodleroot.com/wunderbyte.at
If I add http://wunderbyte.at the url is correct.
If I add https://wunderbyte.at the url is also correct. 

@winnidat is your question why some urls are actually linking to your own moodle?
If yes, the editor tries not to guess what you want, if you add a complete url like https://wunderbyte.at it will send this out. If you add just a part (or a typo like "https:/wunderbyte.at) it assumes you want to link to a site on your own server. This is consistent with html a href links.
